conditional/templates/base.html

```diff
@@ -28,8 +28,9 @@
         </div>
         {% endif %}
 
+        <footer>
+            <a class="footer-version" href="https://github.com/ComputerScienceHouse/conditional">Conditional ({{config["GIT_REVISION"]}})</a>
```

Comment:
The format would be `https://github.com/ComputerScienceHouse/conditional/tree/$commitHash`